Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * SMTChecker: Fix incorrect analysis when only a subset of contracts is selected with `--model-checker-contracts`.
```

Comment:
One last thing. We have this weird inconsistency where some Markdown files use double backticks and some use single ones. Some lists also have this weird one space indent before bullet points. We stuck to it for the sake of consistencym, but it has always been annoying, because people are often confused which one to use where. I'd like to switch to a saner style for the `CHANGELOG` from this release on (https://github.com/ethereum/solidity/pull/15930#discussion_r1994417204).

```suggestion
* SMTChecker: Fix incorrect analysis when only a subset of contracts is selected with `--model-checker-contracts`.
```

Reply:
OK, so, single backticks and no space at the beginning.